lib/AssemblyUtil/AssemblyUtilImpl.py

```diff
@@ -205,7 +204,7 @@ def save_assembly_from_fasta(self, ctx, params):
         ref = FastaToAssembly(
             DataFileUtil(self.callback_url, token=ctx['token']),
             Path(self.sharedFolder)
-        ).import_fasta(params)
+        ).import_fasta(params)['upa']
```

Comment:
so we are returning only `upa` for `save_assembly_from_fasta` but include `filtered_input` for `save_assemblies_from_fastas`.  That's because we want backward compatibility for `save_assembly_from_fasta`?

Reply:
Right. I'm going to make a SAFF2 method that matches the bulk methods output other than returning a list.